src/ui/App.tsx

```diff
@@ -77,27 +79,29 @@ const App = () => {
   }, [authentication.loggedIn, currentRoute]);
 
   useEffect(() => {
+    ScreenOrientation.lock({ orientation: "portrait" });
```

Comment:
I think some things got reverted here post merge. I believe we are only locking on mobile since this plugin doesn't work on web.

Reply:
Yes, you remember right, I made that change here: https://github.com/cardano-foundation/cf-identity-wallet/pull/439/files